src/translation/translation_ko_KR.ts

```diff
@@ -0,0 +1,4164 @@
+        <translation type="vanished">여기에서 설정한 내용은 Jamulus 서버에 연결될 때 믹서 보드의 페이더에 나타납니다. 이 태그는 여러분과 같은 서버에 연결한 클라이언트에도 표시됩니다.</translation>
+    </message>
+    <message>
+        <source>Drum Set</source>
+        <translation type="vanished">드럼 세트</translation>
+    </message>
+    <message>
+        <source>Djembe</source>
+        <translation type="vanished">젬베</translation>
+    </message>
+    <message>
+        <source>Electric Guitar</source>
+        <translation type="vanished">일렉트릭 기타</translation>
+    </message>
+    <message>
+        <source>Acoustic Guitar</source>
+        <translation type="vanished">어쿠스틱 기타</translation>
+    </message>
+    <message>
+        <source>Bass Guitar</source>
+        <translation type="vanished">베이스 기타</translation>
+    </message>
+    <message>
+        <source>Keyboard</source>
+        <translation type="vanished">키보드</translation>
+    </message>
+    <message>
+        <source>Synthesizer</source>
+        <translation type="vanished">신디사이저</translation>
+    </message>
+    <message>
+        <source>Grand Piano</source>
+        <translation type="vanished">그랜드 피아노</translation>
+    </message>
+    <message>
+        <source>Accordion</source>
+        <translation type="vanished">아코디언</translation>
+    </message>
+    <message>
+        <source>Vocal</source>
+        <translation type="vanished">보컬</translation>
+    </message>
+    <message>
+        <source>Microphone</source>
+        <translation type="vanished">마이크</translation>
+    </message>
+    <message>
+        <source>Harmonica</source>
+        <translation type="vanished">하모니카</translation>
+    </message>
+    <message>
+        <source>Trumpet</source>
+        <translation type="vanished">트럼펫</translation>
+    </message>
+    <message>
+        <source>Trombone</source>
+        <translation type="vanished">트롬본</translation>
+    </message>
+    <message>
+        <source>French Horn</source>
+        <translation type="vanished">호른</translation>
+    </message>
+    <message>
+        <source>Tuba</source>
+        <translation type="vanished">튜바</translation>
+    </message>
+    <message>
+        <source>Saxophone</source>
+        <translation type="vanished">색소폰</translation>
+    </message>
+    <message>
+        <source>Clarinet</source>
+        <translation type="vanished">클라리넷</translation>
+    </message>
+    <message>
+        <source>Flute</source>
+        <translation type="vanished">플루트</translation>
+    </message>
+    <message>
+        <source>Violin</source>
+        <translation type="vanished">바이올린</translation>
+    </message>
+    <message>
+        <source>Cello</source>
+        <translation type="vanished">첼로</translation>
+    </message>
+    <message>
+        <source>Double Bass</source>
+        <translation type="vanished">콘트라베이스</translation>
+    </message>
+    <message>
+        <source>Recorder</source>
+        <translation type="vanished"리코더</translation>
```

Comment:
">" missing after "vanished".

Reply:
I found some of them after that I would update at once.

Reply:
If after summitting a correction you see checks at the bottom failing, click on 'Details' and you will see a log . Look for the line starting with 'lrelease error' near the bottom and it'll tell you what line the error is on.

Reply:
Thank you :)

Reply:
It seems ok now.